apis/kubedb/v1alpha2/constants.go

```diff
@@ -347,18 +347,21 @@ const (
 	DatabaseHalted = "Halted"
 
 	// Condition reasons
-	DataRestoreStartedByExternalInitializer = "DataRestoreStartedByExternalInitializer"
-	DatabaseSuccessfullyRestored            = "SuccessfullyDataRestored"
-	FailedToRestoreData                     = "FailedToRestoreData"
-	AllReplicasAreReady                     = "AllReplicasReady"
-	SomeReplicasAreNotReady                 = "SomeReplicasNotReady"
-	DatabaseAcceptingConnectionRequest      = "DatabaseAcceptingConnectionRequest"
-	DatabaseNotAcceptingConnectionRequest   = "DatabaseNotAcceptingConnectionRequest"
-	ReadinessCheckSucceeded                 = "ReadinessCheckSucceeded"
-	ReadinessCheckFailed                    = "ReadinessCheckFailed"
-	DatabaseProvisioningStartedSuccessfully = "DatabaseProvisioningStartedSuccessfully"
-	DatabaseSuccessfullyProvisioned         = "DatabaseSuccessfullyProvisioned"
-	DatabaseHaltedSuccessfully              = "DatabaseHaltedSuccessfully"
+	DataRestoreStartedByExternalInitializer    = "DataRestoreStartedByExternalInitializer"
+	DatabaseSuccessfullyRestored               = "SuccessfullyDataRestored"
+	FailedToRestoreData                        = "FailedToRestoreData"
+	AllReplicasAreReady                        = "AllReplicasReady"
+	SomeReplicasAreNotReady                    = "SomeReplicasNotReady"
+	DatabaseAcceptingConnectionRequest         = "DatabaseAcceptingConnectionRequest"
+	DatabaseNotAcceptingConnectionRequest      = "DatabaseNotAcceptingConnectionRequest"
+	ReadinessCheckSucceeded                    = "ReadinessCheckSucceeded"
+	ReadinessCheckFailed                       = "ReadinessCheckFailed"
+	DatabaseProvisioningStartedSuccessfully    = "DatabaseProvisioningStartedSuccessfully"
+	DatabaseSuccessfullyProvisioned            = "DatabaseSuccessfullyProvisioned"
+	DatabaseHaltedSuccessfully                 = "DatabaseHaltedSuccessfully"
+	InternalUsersReady                         = "InternalUsersReady"
```

Comment:
Type : `InternalUsersReady` --> `InternalUsersSynced`

Reply:
renamed and moved